File: fix_walkthrough.py
import os

path = r'c:\Users\SHIN BAEKMIN\.gemini\antigravity\brain\9c34cb30-b545-435c-b623-860bd528c32b\walkthrough.md'

# Read file (could be UTF-16 from PowerShell's Set-Content)
with open(path, 'rb') as f:
    raw = f.read()

# Detect encoding
if raw[:2] in [b'\xff\xfe', b'\xfe\xff']:
    text = raw.decode('utf-16')
else:
    text = raw.decode('utf-8')

# Replace file:/// URL-encoded paths with absolute paths
old = 'file:///C:/Users/SHIN%20BAEKMIN/.gemini/antigravity/brain/9c34cb30-b545-435c-b623-860bd528c32b/'
new = 'C:/Users/SHIN BAEKMIN/.gemini/antigravity/brain/9c34cb30-b545-435c-b623-860bd528c32b/'
text = text.replace(old, new)

# Write back as UTF-8 without BOM
with open(path, 'w', encoding='utf-8', newline='\n') as f:
    f.write(text)

print('Done - file saved as UTF-8')

File: fix_walkthrough.js
const fs = require('fs');
const path = 'c:/Users/SHIN BAEKMIN/.gemini/antigravity/brain/9c34cb30-b545-435c-b623-860bd528c32b/walkthrough.md';

let raw = fs.readFileSync(path);

// Detect UTF-16 LE BOM
let text;
if (raw[0] === 0xFF && raw[1] === 0xFE) {
  text = raw.toString('utf16le').substring(1); // skip BOM
} else {
  text = raw.toString('utf-8');
  // Remove UTF-8 BOM if present
  if (text.charCodeAt(0) === 0xFEFF) text = text.substring(1);
}

// Replace file:/// URL-encoded paths with simple absolute paths
const oldPath = 'file:///C:/Users/SHIN%20BAEKMIN/.gemini/antigravity/brain/9c34cb30-b545-435c-b623-860bd528c32b/';
const newPath = 'C:/Users/SHIN BAEKMIN/.gemini/antigravity/brain/9c34cb30-b545-435c-b623-860bd528c32b/';

const result = text.split(oldPath).join(newPath);

fs.writeFileSync(path, result, 'utf-8');
console.log('Done - walkthrough.md saved as UTF-8 with fixed image paths');
